### PR TITLE
Ccd 1155

### PR DESCRIPTION
**Pull request by owen-delaney-cgi**
### Change description ###
Updated bulk user import script to handle adding additional roles

**Does this PR introduce a breaking change?** (check one with "x")

```
[x] Yes
[ ] No
```

### CODE REVIEW

File: bulk-user-setup/README.md [hunk 1]
```diff
@@ -24,10 +24,13 @@ The CSV input file must contain the following *mandatory* elements, including a
 
 | Header       | Mandatory | Description                                               |
 |--------------|-----------|-----------------------------------------------------------|
+| operation    | **Yes**   | Either `add` or `update`                                  |
```
Comment (MSancaktutar) [inline bulk-user-setup/README.md:27]: Either `create` or `update`

**Comment by A-R05** [comment]
@owen-delaney-cgi Can you remove the proxy call in the script. Will break the script due to the proxy shutdown and is not needed anymore

**Review by MSancaktutar — CHANGES_REQUESTED** [review]
Please clone the script file into a new one like bulk-user-update.sh.
We'll deliver a new script as part of tis ticket. Later one, this new script will become the script that does both creation and update.

**Review by MSancaktutar — COMMENTED** [review]
README file can still stay as updated. There will be an inconsistency for a short while, but that's acceptable within the current conditions.

File: bulk-user-setup/bulk-user-creation.sh [hunk 2]
```diff
@@ -254,20 +336,47 @@ function process_input_file() {
...
+              combinedRoles=$(echo $currentRoles $rolesToAdd | jq '.[]' | jq -s)
+          fi
+
+          # functionality flakey, removing for now
+          # remove roles to remove from the combined role list
+          # if [ "${rolesToRemove}" != "null" ]; then
+          #   for role in ${rolesToRemove[@]}; do
+          #     combinedRoles=( "${combinedRoles[@]/$role}" )
+          #   done
+          # fi
```
Comment (MSancaktutar) [inline bulk-user-setup/bulk-user-creation.sh:367]: That's fine.

File: bulk-user-setup/sample_test.csv [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
-cepumo@gopwisu.tp,Matilda,Reynolds,caseworker|caseworker-autotest1
-ejeso@men.ci,Adele,Norris,caseworker|caseworker-autotest1|caseworker-autotest1-private
-fobip@fic.va,Alexander,Wagner,caseworker
-luazfi@nihisasi.mx,Eugenia,Hardy,caseworker
+operation,email,firstName,lastName,roles
+add,rejum@tedek.by,Sophie,Owen,caseworker|caseworker-autotest1|caseworker-autotest1-private
+add,cepumo@gopwisu.tp,Matilda,Reynolds,caseworker|caseworker-autotest1
+add,ejeso@men.ci,Adele,Norris,caseworker|caseworker-autotest1|caseworker-autotest1-private
+add,fobip@fic.va,Alexander,Wagner,caseworker
+add,luazfi@nihisasi.mx,Eugenia,Hardy,caseworker
```
Comment (MSancaktutar) [inline bulk-user-setup/sample_test.csv:6]: Change `add` to `create` please.

File: bulk-user-setup/README.md (hunk 1 above)
Comment (owen-delaney-cgi) [reply bulk-user-setup/README.md]: Updated

File: bulk-user-setup/sample_test.csv (hunk 3 above)
Comment (owen-delaney-cgi) [reply bulk-user-setup/sample_test.csv]: Updated

**Review by MSancaktutar — APPROVED** [review]
Didn't check in lower level details, but it looks to be fit for purpose and good to go for the immediate ticket. Later tweaks can be done under the followup ticket. Thank you.

**Pull request merged**